src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -117,12 +117,12 @@ const submitEncryptModeForm: ControllerHandler<
       meta: logMeta,
       error: formPublicResult.error,
     })
-    const { statusCode } = mapRouteError(formPublicResult.error)
+    const { statusCode, errorMessage } = mapRouteError(formPublicResult.error)
     if (statusCode === StatusCodes.GONE) {
       return res.sendStatus(statusCode)
     } else {
       return res.status(statusCode).json({
-        message: form.inactiveMessage,
+        message: errorMessage || form.inactiveMessage,
```

Comment:
I think we can remove the form.inactiveMessage part; there will always be an errorMessage